handlers/tokeninfo/proxy/handler.go

```diff
@@ -0,0 +1,25 @@
+package tokeninfoproxy
+
+import (
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"github.com/zalando/planb-tokeninfo/handlers/tokeninfo"
+)
+
+type tokenInfoProxyHandler struct {
+	upstream *httputil.ReverseProxy
+}
+
+func (h *tokenInfoProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	token := req.URL.Query().Get("access_token")
```

Comment:
same here: we need to support `Authorization` (preferred as it will/should never be logged in libraries!) , see https://github.com/zalando/planb-tokeninfo/commit/204e94502df4738e8a981d556ef0b3fb5e62955a